app/controllers/new_feature_controller.rb

```diff
@@ -0,0 +1,8 @@
+class NewFeatureController < Devise::SessionsController
+  def hoge
+    a = 1  
+    b = 1
+    c = 3
+    d = a
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Lint/UselessAssignment: Useless assignment to variable - d.